translate/src/main/java/com/google/cloud/translate/samples/TranslateText.java

```diff
@@ -61,13 +61,14 @@ public static void translateText(String sourceText, PrintStream out) {
 
   /**
    * Translate the source text from source to target language.
+   * Make sure that your project is whitelisted. 
    *
    * @param sourceText source text to be translated
    * @param sourceLang source language of the text
    * @param targetLang target language of translated text
    * @param out print stream
    */
-  public static void translateTextWithOptions(
+  public static void translateTextWithOptionsAndModel(
```

Comment:
I guess I was trying to clearly demarcate WL & non-WL users and so to avoid confusion? Sorry did quite last minute so may not have a very good explanation.